docs/examples/aws_ebs_volume_idempotence.rb

```diff
@@ -0,0 +1,34 @@
+require 'chef/provisioning/aws_driver'
+
+with_driver 'aws::us-west-2'
+
+key = aws_key_pair 'ref-key-pair-ebs'
+key.run_action(:create)
+
+ref_machine1 = machine 'ref-machine-1'
+ref_machine1.run_action(:converge)
+
+ebs_volume = aws_ebs_volume 'ref-volume-ebs' do
+  action :nothing
+  machine ref_machine1.name
+  device '/dev/xvdf'
+  availability_zone 'us-west-2a'
+  size 1
+end
+
+ebs_volume.run_action(:create)
+ebs_volume.run_action(:create) # up to date
+ebs_volume.run_action(:attach)
+ebs_volume.run_action(:attach) # up to date
+ebs_volume.run_action(:detach)
+ebs_volume.run_action(:detach) # up to date
+ebs_volume.run_action(:delete)
+ebs_volume.run_action(:delete) # up to date
```

Comment:
Before all the recent provisioning changes, only the first compiled resource would be executed.  However, I just tried it from master and it worked!  I'll b make the changes to the other tests as well.